packages/types/src/clerk.ts

```diff
@@ -996,6 +1001,11 @@ export type OrganizationSwitcherProps = CreateOrganizationMode &
      * the number of max allowed members is equal to 1
      */
     skipInvitationScreen?: boolean;
+    /**
+     * Hides the slug field in the form.
+     * @default undefined
```

Comment:
Shouldn't this be `@default false` ?

Reply:
By default it has no value that's why default is `undefined`, but I got your point 👍🏼

Reply:
Updated the default values in runtime to `false` and prop types to `false` 👍🏼

Reply:
Correct, since we are not treating undefined as a third state we can state `false`.